odoo/models.py

```diff
@@ -2667,13 +2667,17 @@ def _add_inherited_fields(self):
                 #  - reading inherited fields should not bypass access rights
                 #  - copy inherited fields iff their original field is copied
                 Field = type(field)
+                field_attrs = {}
+                if field.type == 'properties':
+                    field_attrs['definition'] = field.definition
```

Comment:
Should this part be in fields.py ? `Fields.setup_related` method looks like a better place to do that (also will manage the standard related case) ?

Reply:
Hello @ryv-odoo ! You are right it seems a better place indeed, sorry.

As for the standard related case, i'm not sure what to do. The current definition implementation does not really work with "multiple dots" paths. You can even find a `assert self.definition.count(".") == 1` to forbid it. I'm not sure how big of a change it is to allow this.

Reply:
Never mind about standard related then.
Thank you for the change.

Reply:
Ok! Is the PR ok then or is something missing?